js/data/locale/en/GB/unitfmt.json

```diff
@@ -44,7 +44,7 @@
             "dessert-spoon": "one#{n} US dessertspoon|#{n} US dessertspoons",
             "dessert-spoon-imperial": "one#{n} dessertspoon|#{n} dessertspoons",
             "dot": "one#{n} dot|#{n} dots",
-            "dot-per-centimeter": "one#{n} dot per centimetre|#{n} dots per centimetre",
+            "dot-per-centimeter": "one#{n} dot per centimeter|#{n} dots per centimeter",
```

Comment:
This is definitely not right. British spelling is "centimetre" like it was before. Are you sure this is what it says in CLDR?

Reply:
Yes It's changes from cldr v44.1 (cldr-units-full/main/en-GB/units.json)
In units.json of CLDR, it has `area-square-centimeter`, `graphics-pixel-per-centimeter`, `graphics-dot-per-centimeter`, `length-centimeter`, and `volume-cubic-centimeter`. 
others still have `centimetre` but only `graphics-dot-percentimeter` value has changed t `centimeter` in this version.

Reply:
So stupid... why don't they have integrity checks? Spelling it one way in one unit and a different way in another unit is not a preferential thing. Anyways, we will have to wait until a future version of CLDR to get it corrected, I supposed.